system-test/datastore.ts

```diff
@@ -861,34 +859,32 @@ describe('Datastore', () => {
               },
             ]);
 
-            transaction.commit(err => {
+            transaction.commit(async err => {
```

Comment:
Having an async callback method is super weird.  I believe `transaction.commit` is an async function.  Can you just `await` the result of that?